shaders/src/main/glsl/samples/310es/selection_sort_struct.frag

```diff
@@ -0,0 +1,77 @@
+#version 310 es
+
+/*
+ * Copyright 2019 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+precision highp float;
+
+layout(location = 0) out vec4 _GLF_color;
+
+uniform vec2 injectionSwitch;
+
+uniform vec2 resolution;
+
+struct Obj {
+  float odd_numbers[10];
+  float even_numbers[10];
+};
+
+void main() {
+  Obj obj;
+  
+  // Initialize first 10 odd numbers to the array 
+  int odd_index = 0;
+  float odd_number = 1.;
+  while(odd_index <= 10){
+    obj.odd_numbers[odd_index] = odd_number;
+    odd_number += 2.;
+    odd_index++;
+  }
```

Comment:
I think there is an out of bound access here: `obj.odd_numbers[odd_index]` where `odd_index` is 10.

Reply:
Thank you! Fixed